src/com/company/Flowers/Chamomile.java

```diff
@@ -0,0 +1,17 @@
+package com.company.Flowers;
+
+public class Chamomile extends Flower {
+
+    public Chamomile() {
+        price = 70;
```

Comment:
for this case it is ok, but better to pass value as parameter instead of using "magic numbers"

Reply:
Thanks, I understand, will try to avoid this approach.